### DISCUSSION TITLE
Rulesets not properly loading (Android)

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#16941] [Q&A] [answered]
Forgive typing on mobile, anyhoo I decided to try playing yoso on my phone but after copying over the dll to the rulesets folder and launching the game I get errors for the outdated rulesets still present in the directory indicating it is finding them and attempting to load rulesets but no new modes show in the top bar and i only get a few errors not matching the amount of rulesets in the directory and none mentioning the yoso ruleset I wanted to try out. Going to to attach a screenshot of the errors showing only that yoso is not one of them as the errors do not pertain to the problem at hand as I get them on my pc as well just borked rulesets. Please let me know what other information you would like from me.

![](https://i.imgur.com/um5L1ar.jpg)

**Comment by peppy** [comment, marked as answer]
Custom rulesets will not work on mobile platforms for now.
